packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -53,6 +53,8 @@ export class Popup extends React.Component<IPopupProps, IPopupState> {
 
   public componentWillUnmount(): void {
     this._disposables.forEach((dispose: () => void) => dispose());
+    this._async.dispose();
```

Comment:
I think you can just add async to disposables and that's all you need to do. I don't think having a reference to the element would keep things alive.

Reply:
It is the closure in the `this._disposables` array that is retaining the DOM element so `delete this._disposables` or what you recommended should suffice. I'll revise and verify in a few.

Reply:
In my testing we need to explicitly de-reference `this._originalFocusedElement` because it gets captured via `_this` in the closure of the transpiled `_onKeyDown` function.

![image](https://user-images.githubusercontent.com/706967/71872503-7eb04200-30d1-11ea-956c-b223675ceff6.png)

![image](https://user-images.githubusercontent.com/706967/71872531-925ba880-30d1-11ea-9d21-659bbde8f1ba.png)

I've reverted my changes to reflect a fix for the above.